src/core/infrastructure/BaseService.ts

```diff
@@ -14,7 +14,7 @@ export interface BaseServiceOptions {
   jobToken?: string;
   host?: string;
   url?: string;
-  version?: 'v3' | 'v4';
+  version?: 3 | 4;
```

Comment:
This seems breaking 🤔

Reply:
Yea i mentioned it in the commit messages!

Reply:
yet major version wasn't bumped 😆

Reply:
Should have used rebase and merge

Reply:
yea im fixing it now